libs/platform/ebpf_epoch.c

```diff
@@ -266,7 +230,10 @@ ebpf_epoch_exit()
     if (ebpf_is_preemptible()) {
```

Comment:
move the code to get current_cpu etc. exclusively in the non-preemptive branch?

Reply:
It's used in both branches. In the preemptible case, it's used to pick a CPU to pick an ebpf_epoch_cpu_entry_t.